packages/components/buttons/icon-button/src/icon-button.styles.ts

```diff
@@ -155,13 +158,15 @@ const getBaseStyles = (
       `;
     case 'info':
       return css`
+        background-color: ${designTokens.colorInfo};
+
         &:hover {
-          background-color: ${designTokens.colorInfo};
+          background-color: ${designTokens.colorInfo40};
           box-shadow: ${designTokens.shadow0};
         }
         ${isActive ? '&,' : ''}
         &:active {
-          background-color: #057fcc;
+          background-color: ${designTokens.colorInfo};
```

Comment:
@FilPob  I implemented it as shown in figma, but since it uses the same color for the default and active state, a  `IconButton` with `info` theme can not be used as a toggle button as it wouldn't give any visual feedback.

Reply:
Oh I see... the toggled state for the info IconButton should have the background-color: `color-info-40` will correct that also in Figma.

Reply:
`color-info-40` is currently also used in the hover-state. I'm happy to change it, but checking the updated figma screen, the hover-color user there is an undocumented color (`#089BF7`) and not referring to a token.

Reply:
Yes exactly. `color-info-50` for the hover state. Thanks